lib/at_cmd_parser/at_utils.h

```diff
@@ -242,6 +242,30 @@ static inline bool is_command(const char *str)
 	return false;
 }
 
+/**
+ * @brief Check if a string is a beginning of an AT CLAC response
+ *
+ * This function will check if the string is a CLAC response prefix.
+ * Valid prefixes: AT+ and AT%, except AT%X
```

Comment:
Why is `AT%X` an exception?

Reply:
There are some responses starting with AT%X that are not CLAC responses: Read response for XCOEX0, XDATAPRFL, XMAGPIO, XOPCONF, and test response for XDEEPSEARCH. They could be falsely processed as CLAC. 
This logic assumes none of these will be the first command listed in the CLAC response, which seems like a good bet.